packages/core/src/amazonqFeatureDev/client/featureDev.ts

```diff
@@ -46,6 +46,7 @@ export async function createFeatureDevProxyClient(options?: Partial<ServiceOptio
             region: cwsprConfig.region,
             endpoint: cwsprConfig.endpoint,
             token: new Token({ token: bearerToken }),
+            customUserAgent: getUserAgent(),
```

Comment:
this is already done by `createAwsService()`, which is called on line 42 above.

Reply:
is a caller to `createFeatureDevProxyClient` perhaps passing `customUserAgent` and overriding this?

Reply:
I can not see where `customUserAgent` predefined in `createAwsService`. Can you point me to the right place?

I have verified there is not callers override the `customUserAgent`.

Reply:
https://github.com/aws/aws-toolkit-vscode/blob/5573cbb1f56f9ec38a1c20572113eda1e5556d90/packages/core/src/shared/awsClientBuilder.ts#L135-L137

Reply:
Thanks, I am going to cancel this PR then.